data.sql

```diff
@@ -4,3 +4,13 @@ INSERT INTO animals(name, date_of_birth, escape_attempts, neutered, weight_kg) V
 iNSERT INTO animals(name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Gabumon', '2018-11-15', 2, true, 8), ('Pikachu', '2021-01-07', 1, false, 15.04), ('Devimon', '2
 017-05-12', 5, true, 11);
 INSERT INTO animals(name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Charmander', '2020-02-08', 0, false, -11), ('Plantmon', '2021-11-15', 2, true, -5.7), ('Squirtle', '1993-04-02', 3, false, -12.13), ('Angemon', '2005-06-12', 1, true, -45), ('Boarmon', '2005-06-07', 7, true, 20.4), ('Blossom', '1998-10-13', 3, true, 17), ('Ditto', '2022-05-14', 4, true, 22);
+
+-- querry multiple tables
+INSERT INTO owners (full_name, age) VALUES ('Sam Smith', 34), ('Jennifer Orwell', 19), ('Bob', 45), ('Melody Pond', 77), ('Dean Winchester', 14), ('Jodie Whittaker', 38);
+INSERT INTO species (name) VALUES ('Pokemon'), ('Digimon');
+UPDATE animals SET species_id = 2 WHERE name LIKE'%mon';
```

Comment:
:orange_circle: [OPTIONAL] You could also use something like this :arrow_down: 
```
  UPDATE animals
   SET species_id = (SELECT id from species WHERE name = 'Digimon')
   WHERE name like '%mon';
```
I prefer this one because it is more dynamic. If you want to change the species_id value, you won't need to update the query which is not the case when you hardcoded species_id.

Reply:
Thank you :handshake: